From 6cd83061a0f5eccfff9c39be873106ddb9434851 Mon Sep 17 00:00:00 2001
From: "Pan, Kris" <kris.pan@intel.com>
Date: Tue, 25 Aug 2020 15:43:17 +0800
Subject: [PATCH 1205/1214] ASoC: Intel: add more dependency for skylake virtio
 fe driver

Error Log :
WARNING: sound/soc/intel/skylake/virtio/snd-soc-skl-virtio-fe:
'kctl_notify_machine_ready' exported twice.
Previous export was in sound/soc/intel/skylake/virtio/snd-soc-skl-virtio-be.ko

Signed-off-by: Pan, Kris <kris.pan@intel.com>
---
 sound/soc/intel/skylake/virtio/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/skylake/virtio/Kconfig b/sound/soc/intel/skylake/virtio/Kconfig
index 0926781..1b2b1a4 100644
--- a/sound/soc/intel/skylake/virtio/Kconfig
+++ b/sound/soc/intel/skylake/virtio/Kconfig
@@ -1,6 +1,6 @@
 config SND_SOC_INTEL_SKYLAKE_VIRTIO_FE
 	tristate "SKL/BXT/KBL/GLK/CNL... Platforms Virtio FrontEnd"
-	depends on ACRN_VIRTIO_DEVICES && SND_SOC_INTEL_SKYLAKE
+	depends on ACRN_VIRTIO_DEVICES && SND_SOC_INTEL_SKYLAKE && SND_SOC_INTEL_SKYLAKE_VIRTIO_BE=n
 	help
 	  Enables virtio frontend for Intel Skylake audio driver.
 	  This option should be enabled only on Guest OS.
-- 
2.7.4

